test/tools/yulInterpreter/Interpreter.h

```diff
@@ -243,6 +279,7 @@ class ExpressionEvaluator: public ASTWalker
 	std::vector<u256> m_values;
 	/// Current expression nesting level
 	unsigned m_nestingLevel = 0;
+	bool m_enableExternalCalls;
```

Comment:
As stated above, maybe its better to either have always "enable*" or always "disable*"